packages/create-turbo/templates/berry/README.md

```diff
@@ -0,0 +1,73 @@
+# Turborepo starter
+
+This is an official Yarn (Berry) starter turborepo.
+
+## What's inside?
+
+This turborepo uses [Yarn](https://yarnpkg.com/) as a package manager. It includes the following packages/apps:
+
+### Apps and Packages
+
+- `docs`: a [Next.js](https://nextjs.org) app
+- `web`: another [Next.js](https://nextjs.org) app
+- `ui`: a stub React component library shared by both `web` and `docs` applications
+- `eslint-config-custom`: `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
+- `tsconfig`: `tsconfig.json`s used throughout the monorepo
+
+Each package/app is 100% [TypeScript](https://www.typescriptlang.org/).
+
+### Utilities
+
+This turborepo has some additional tools already setup for you:
+
+- [TypeScript](https://www.typescriptlang.org/) for static type checking
+- [ESLint](https://eslint.org/) for code linting
+- [Prettier](https://prettier.io) for code formatting
+
+### Build
+
+To build all apps and packages, run the following command:
+
+```
+cd my-turborepo
+yarn run build
+```
+
+### Develop
+
+To develop all apps and packages, run the following command:
+
+```
+cd my-turborepo
+yarn run dev
+```
+
+### Remote Caching
+
+Turborepo can use a technique known as [Remote Caching](https://turborepo.org/docs/core-concepts/remote-caching) to share cache artifacts across machines, enabling you to share build caches with your team and CI/CD pipelines.
+
+By default, Turborepo will cache locally. To enable Remote Caching you will need an account with Vercel. If you don't have an account you can [create one](https://vercel.com/signup), then enter the following commands:
+
+```
+cd my-turborepo
+yarn dlx turbo login
```

Comment:
This will download and execute the latest `turbo` version from the registry, shouldn't it instead use the version installed in the project?

Reply:
`npx` at least is a _little_ bit more-clever than this. From the `npx` docs:

> By default, npx will check whether <command> exists in $PATH, or in the local project binaries, and execute that. If <command> is not found, it will be installed prior to execution.

I haven't carefully reviewed the best method of executing things for each package manager but if you would like to do the research we would absolutely accept a PR to update the documentation for each. For example: `pnpm -- turbo login` should work, but I don't know about v5/v6/v7 differences in that invocation.

Reply:
```suggestion
yarn turbo login
```
Since `turbo` is a dependency already using `yarn turbo` would be correct in this case.